[2023-10-27 14:16:40,940] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 14:16:40,947] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 14:16:40,948] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:16:40,948] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 14:16:40,948] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:16:40,962] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-06 00:00:00+00:00
[2023-10-27 14:16:40,966] {standard_task_runner.py:52} INFO - Started process 17280 to run task
[2023-10-27 14:16:40,970] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-06T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmp_eg12qrb', '--error-file', '/tmp/tmpe3qu113u']
[2023-10-27 14:16:40,971] {standard_task_runner.py:80} INFO - Job 19: Subtask GOOG
[2023-10-27 14:16:41,019] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 14:16:41,073] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-06T00:00:00+00:00
[2023-10-27 14:16:41,568] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/decorators/base.py", line 179, in execute
    return_value = super().execute(context)
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/gabriel/Documents/project_finance/airflow/dags/get_finance.py", line 18, in get_history
    yfinance.Ticker(ticker).history(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/core/generic.py", line 3902, in to_csv
    return DataFrameRenderer(formatter).to_csv(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/format.py", line 1152, in to_csv
    csv_formatter.save()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/csvs.py", line 247, in save
    with get_handle(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/common.py", line 739, in get_handle
    check_parent_directory(str(handle))
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/common.py", line 604, in check_parent_directory
    raise OSError(rf"Cannot save file into a non-existent directory: '{parent}'")
OSError: Cannot save file into a non-existent directory: '/home/gabriel/Documents/project_finance/datalake/stocks/GOOG'
[2023-10-27 14:16:41,574] {taskinstance.py:1395} INFO - Marking task as FAILED. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231006T000000, start_date=20231027T171640, end_date=20231027T171641
[2023-10-27 14:16:41,588] {standard_task_runner.py:92} ERROR - Failed to execute job 19 for task GOOG (Cannot save file into a non-existent directory: '/home/gabriel/Documents/project_finance/datalake/stocks/GOOG'; 17280)
[2023-10-27 14:16:41,598] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-10-27 14:16:41,614] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 14:29:53,784] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 14:29:53,801] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 14:29:53,802] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:29:53,802] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 14:29:53,802] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:29:53,821] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-06 00:00:00+00:00
[2023-10-27 14:29:53,827] {standard_task_runner.py:52} INFO - Started process 19646 to run task
[2023-10-27 14:29:53,836] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-06T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmp8vp80un8', '--error-file', '/tmp/tmpkrse1hbw']
[2023-10-27 14:29:53,837] {standard_task_runner.py:80} INFO - Job 19: Subtask GOOG
[2023-10-27 14:29:53,916] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 14:29:54,153] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-06T00:00:00+00:00
[2023-10-27 14:29:54,947] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/decorators/base.py", line 179, in execute
    return_value = super().execute(context)
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/gabriel/Documents/project_finance/airflow/dags/get_finance.py", line 20, in get_history
    yfinance.Ticker(ticker).history(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/core/generic.py", line 3902, in to_csv
    return DataFrameRenderer(formatter).to_csv(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/format.py", line 1152, in to_csv
    csv_formatter.save()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/csvs.py", line 247, in save
    with get_handle(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/common.py", line 863, in get_handle
    handle = open(
PermissionError: [Errno 13] Permission denied: '/GOOG_20231006.csv'
[2023-10-27 14:29:54,955] {taskinstance.py:1395} INFO - Marking task as FAILED. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231006T000000, start_date=20231027T172953, end_date=20231027T172954
[2023-10-27 14:29:54,998] {standard_task_runner.py:92} ERROR - Failed to execute job 19 for task GOOG ([Errno 13] Permission denied: '/GOOG_20231006.csv'; 19646)
[2023-10-27 14:29:55,025] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-10-27 14:29:55,068] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 14:33:44,455] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 14:33:44,475] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 14:33:44,475] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:33:44,475] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 14:33:44,475] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:33:44,490] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-06 00:00:00+00:00
[2023-10-27 14:33:44,497] {standard_task_runner.py:52} INFO - Started process 20607 to run task
[2023-10-27 14:33:44,507] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-06T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmp_lzpy3ta', '--error-file', '/tmp/tmptei8_57g']
[2023-10-27 14:33:44,508] {standard_task_runner.py:80} INFO - Job 19: Subtask GOOG
[2023-10-27 14:33:44,582] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 14:33:44,722] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-06T00:00:00+00:00
[2023-10-27 14:33:45,214] {python.py:173} INFO - Done. Returned value was: None
[2023-10-27 14:33:45,221] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231006T000000, start_date=20231027T173344, end_date=20231027T173345
[2023-10-27 14:33:45,296] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-27 14:33:45,325] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 14:54:03,783] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 14:54:03,789] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 14:54:03,789] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:54:03,789] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 14:54:03,789] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:54:03,798] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-06 00:00:00+00:00
[2023-10-27 14:54:03,801] {standard_task_runner.py:52} INFO - Started process 23306 to run task
[2023-10-27 14:54:03,804] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-06T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmpee9fbwkt', '--error-file', '/tmp/tmpxd48wu7j']
[2023-10-27 14:54:03,805] {standard_task_runner.py:80} INFO - Job 19: Subtask GOOG
[2023-10-27 14:54:03,840] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 14:54:03,883] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-06T00:00:00+00:00
[2023-10-27 14:54:04,341] {python.py:173} INFO - Done. Returned value was: None
[2023-10-27 14:54:04,347] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231006T000000, start_date=20231027T175403, end_date=20231027T175404
[2023-10-27 14:54:04,389] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-27 14:54:04,404] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 15:23:18,859] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 15:23:18,875] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 15:23:18,875] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 15:23:18,875] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 15:23:18,876] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 15:23:18,885] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-06 00:00:00+00:00
[2023-10-27 15:23:18,888] {standard_task_runner.py:52} INFO - Started process 27160 to run task
[2023-10-27 15:23:18,891] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-06T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmps3uexkf_', '--error-file', '/tmp/tmpn0dwvpp0']
[2023-10-27 15:23:18,891] {standard_task_runner.py:80} INFO - Job 19: Subtask GOOG
[2023-10-27 15:23:18,927] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 15:23:19,007] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-06T00:00:00+00:00
[2023-10-27 15:23:19,440] {python.py:173} INFO - Done. Returned value was: None
[2023-10-27 15:23:19,445] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231006T000000, start_date=20231027T182318, end_date=20231027T182319
[2023-10-27 15:23:19,491] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-27 15:23:19,507] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 17:01:44,113] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 17:01:44,120] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 17:01:44,120] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 17:01:44,120] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 17:01:44,120] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 17:01:44,132] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-06 00:00:00+00:00
[2023-10-27 17:01:44,136] {standard_task_runner.py:52} INFO - Started process 39360 to run task
[2023-10-27 17:01:44,139] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-06T00:00:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmpdmn87p0i', '--error-file', '/tmp/tmpxy3j9tcc']
[2023-10-27 17:01:44,139] {standard_task_runner.py:80} INFO - Job 22: Subtask GOOG
[2023-10-27 17:01:44,176] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 17:01:44,221] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-06T00:00:00+00:00
[2023-10-27 17:01:44,724] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/decorators/base.py", line 179, in execute
    return_value = super().execute(context)
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/gabriel/Documents/project_finance/airflow/dags/get_finance.py", line 20, in get_history
    yfinance.Ticker(ticker).history(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/core/generic.py", line 3902, in to_csv
    return DataFrameRenderer(formatter).to_csv(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/format.py", line 1152, in to_csv
    csv_formatter.save()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/csvs.py", line 247, in save
    with get_handle(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/common.py", line 863, in get_handle
    handle = open(
IsADirectoryError: [Errno 21] Is a directory: '/home/gabriel/Documents/project_finance/datalake/Bronze/stocks/GOOG'
[2023-10-27 17:01:44,733] {taskinstance.py:1395} INFO - Marking task as FAILED. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231006T000000, start_date=20231027T200144, end_date=20231027T200144
[2023-10-27 17:01:44,756] {standard_task_runner.py:92} ERROR - Failed to execute job 22 for task GOOG ([Errno 21] Is a directory: '/home/gabriel/Documents/project_finance/datalake/Bronze/stocks/GOOG'; 39360)
[2023-10-27 17:01:44,788] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-10-27 17:01:44,804] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 17:08:55,691] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 17:08:55,698] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 17:08:55,698] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 17:08:55,698] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 17:08:55,698] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 17:08:55,708] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-06 00:00:00+00:00
[2023-10-27 17:08:55,712] {standard_task_runner.py:52} INFO - Started process 41070 to run task
[2023-10-27 17:08:55,716] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-06T00:00:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmp36m745e_', '--error-file', '/tmp/tmp_kqr440k']
[2023-10-27 17:08:55,717] {standard_task_runner.py:80} INFO - Job 22: Subtask GOOG
[2023-10-27 17:08:55,796] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 17:08:56,036] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-06T00:00:00+00:00
[2023-10-27 17:08:56,545] {python.py:173} INFO - Done. Returned value was: None
[2023-10-27 17:08:56,557] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231006T000000, start_date=20231027T200855, end_date=20231027T200856
[2023-10-27 17:08:56,597] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-27 17:08:56,651] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 18:09:49,070] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 18:09:49,081] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [queued]>
[2023-10-27 18:09:49,082] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 18:09:49,082] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 18:09:49,082] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 18:09:49,100] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-06 00:00:00+00:00
[2023-10-27 18:09:49,105] {standard_task_runner.py:52} INFO - Started process 49123 to run task
[2023-10-27 18:09:49,110] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-06T00:00:00+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmp2rv5av4l', '--error-file', '/tmp/tmpf2174cjp']
[2023-10-27 18:09:49,111] {standard_task_runner.py:80} INFO - Job 21: Subtask GOOG
[2023-10-27 18:09:49,266] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-06T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 18:09:49,386] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-06T00:00:00+00:00
[2023-10-27 18:09:50,104] {python.py:173} INFO - Done. Returned value was: None
[2023-10-27 18:09:50,121] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231006T000000, start_date=20231027T210949, end_date=20231027T210950
[2023-10-27 18:09:50,154] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-27 18:09:50,192] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
